.github/workflows/amd64.Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM arm64v8/golang:1.23
+FROM golang:1.23-alpine
+
+# Install GCC and musl-dev (needed for C libraries)
```

Comment:
Nit: Don't think this is necessary, but if it is deemed necessary then refocus on Why not What 🙂 
```suggestion
```

Reply:
I think its due to this package from memory: `github.com/mattn/go-sqlite3`

Reply:
For clarity: My review comment is about the comment. Not about the apk add.
Not sure if that was unclear, from @shazbert's reply.

Reply:
Yep that's clear 👍, fixed in https://github.com/thrasher-corp/gocryptotrader/pull/1794/commits/02f3c65ff23056aaa469421fb71d7b863d89fba8